packages/twenty-server/src/engine/twenty-orm/twenty-orm-core.module.ts

```diff
@@ -5,12 +5,16 @@ import {
   Module,
   OnApplicationShutdown,
   Provider,
+  Type,
 } from '@nestjs/common';
 import {
   ConfigurableModuleClass,
   MODULE_OPTIONS_TOKEN,
 } from '@nestjs/common/cache/cache.module-definition';
 
+import { importClassesFromDirectories } from 'typeorm/util/DirectoryExportedClassesLoader';
+import { Logger as TypeORMLogger } from 'typeorm/logger/Logger';
```

Comment:
Not really pretty, but I think it's better than just rewriting this functions internally.